This patch forces the addition of local changes (e.g., removal of `/bin/pwd'
from `configure') to the tarball.

diff --git a/Makefile b/Makefile
index 61cfa43..a6e7cee 100644
--- a/Makefile
+++ b/Makefile
@@ -363,6 +363,10 @@ dist-prepare: $(files-for-dist)
 
 %.tar: FORCE
 	git archive --prefix=$*/ $* > $@.new
+	tar -rv --owner=0 --group=0		\
+	  --transform='s|^\(.*\)$$|$*/\1|g'	\
+	  -f "$@.new"				\
+	  `git diff --name-only`
 	mv -f $@.new $@
 
 # Do `make dist dist-version=X.Y.Z' to make tar files of an older version.
